### PR TITLE
Feature/form restructure

### PR DESCRIPTION
**Pull request by itsnameissteven**
## Is this a fix or a feature?
This is a feature
## What is the change?
* Form was refactored inside another component
* Time input added
* Days of week checkboxes added
* General styles added
## What does it fix?
n/a
## Where should the reviewer start?
Review the new structure of form, 
Review new sass

### CODE REVIEW

File: src/components/Registration/Registration.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    transform: `scaleX(.${page * 20})`
+  };
+
+  const handleSubmit = (e:FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    setPage(page + 1)
+  }
+
+  return(
+    <form onSubmit={e => handleSubmit(e)} className="registration">
```
Comment (alexthompson207) [inline src/components/Registration/Registration.tsx:24]: Would we want to use router here or does conditional rendering make more sense?

File: src/interfaces/interfaces.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  destination: string;
+  setDestination: Dispatch<SetStateAction<string>>;
+  departTime: string;
+  setDepartTime: Dispatch<SetStateAction<string>>;
+  days: DayPicker;
+  setDays: Dispatch<SetStateAction<DayPicker>>;
+  password: string;
+  setPassword: Dispatch<SetStateAction<string>>;
+  bio: string;
+  setBio: Dispatch<SetStateAction<string>>;
```
Comment (alexthompson207) [inline src/interfaces/interfaces.tsx:29]: This is a beast!

**Review by alexthompson207 — APPROVED** [review]
Awesome work with this, maybe in the future we can do some refactoring so app can hold less state.

File: src/components/Registration/Registration.tsx (hunk 1 above)
Comment (PaigeVannelli) [reply src/components/Registration/Registration.tsx:24]: Yeah maybe there's a chance to refactor here!

**Review by PaigeVannelli — APPROVED** [review]

**Pull request merged**